initial-data-load/src/mapLayers/layerGeoDataSchema.js

```diff
@@ -1,10 +1,41 @@
 const mongoose = require('mongoose');
 
+const FeatureIdsSchema = new mongoose.Schema(
+  {
+    property: String,
+    values: [{ type: String }],
+  },
+  { _id: false },
+);
+
+const GeoMetadataSchema = new mongoose.Schema(
```

Comment:
Do we want GeoMetadataSchema to look like this? Or it should be just Object? The same in the api.

Reply:
for now we leave it like this as things for it are in use, its content will be redefined later